crates/toolchain/build/src/lib.rs

```diff
@@ -162,25 +165,27 @@ pub fn cargo_command(subcmd: &str, rust_flags: &[&str]) -> Command {
     println!("Using rustc: {rustc}");
 
     let mut cmd = sanitized_cmd("cargo");
-    // TODO[jpw]: remove +nightly
+    let toolchain = format!("+{RUSTUP_TOOLCHAIN_NAME}");
```

Comment:
can you move this above L158 and reuse?

Reply:
@jonathanpwang Thanks for your review. Two review suggestions are applied at [bb41c53](https://github.com/openvm-org/openvm/pull/1124/commits/bb41c53bc3552de5c359473d721c8b14d6476da9) , PTAL